addons/analytic/tests/test_analytic_account.py

```diff
@@ -217,6 +217,11 @@ def test_analytic_account_branches(self):
         """
         Test that an analytic account defined in a parent company is accessible in its branches (children)
         """
+        # timesheet adds a rule to forcer a project_id; account overrides it
+        timesheet_group = self.env.ref('hr_timesheet.group_hr_timesheet_user', raise_if_not_found=False)
+        if timesheet_group:
+            self.env.user.groups_id -= timesheet_group
```

Comment:
```suggestion
            self.env.user.write({'groups_id': [Command.unlink(timesheet_group.id)]})
```

Reply:
I don't agree with this suggestion, the current approach does exactly the same but is shorter and easier to write.